packages/docs-app/src/examples/core-examples/tabsExample.tsx

```diff
@@ -27,6 +27,7 @@ import {
     Switch,
     Tab,
     type TabId,
+    TabPanel,
```

Comment:
could we have this component show up on the docs page similar to how `Menu item` shows up under `Menu`?

Reply:
Done